intelsdi-x/snap-plugin-collector-osv
# Fix zero values in Grafana and config problem

Summary of changes:
- Replace Godeps with Glide
- Fix zero values in Grafana (return values in int64 instead of string)
- Update config keys in documentation

How to verify it:
- Load plugin
- Create task with InfluxDB publisher
- Check InfluxDB measurement values

Testing done:
- Manual testing
